apps/prairielearn/elements/pl-drawing/defaults.py

```diff
@@ -1,4 +1,47 @@
-element_defaults = {
+from typing import TypedDict
+
+ElementDefaults = TypedDict(
```

Comment:
Can you define this type using the class definition style?: https://peps.python.org/pep-0589/

Reply:
No, it has hyphens in some properties.